test/test_parsing.rb

```diff
@@ -43,6 +43,10 @@ def test_handle_generic
 
     assert_nil Chronic.parse("1/1/32.1")
 
+    time = Chronic.parse("2020-11-01 00:00:00")
```

Comment:
This tests your PR only when system's timezone is one you have, I have different one so for me this test passes either way even without your PR. Need to write such test that would test your specific case with your timezone.